net-misc/streamlink/streamlink-0.12.1.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Why are you adding the old version? Is there any reason anyone would use it instead of 0.13.0?

Reply:
I didn't know the convention regarding that. I'll remove it.

Reply:
The idea is that if you add two versions, people will just upgrade to the newer and the older will never be really used.

Reply:
Fixed.